pdf/pdfdocument.cpp

```diff
@@ -162,6 +217,16 @@ void PDFDocument::requestPage(int index, int size, QQuickWindow *window, QRect s
     d->thread->queueJob(job);
 }
 
+void PDFDocument::requestArea(int index, int size, QQuickWindow *window, QRect subpart)
+{
+    if (!isLoaded() || isLocked() || subpart.isEmpty())
+        return;
+
+    RenderPageJob* job = new RenderPageJob(index, size, window, subpart,
+                                           RenderPageJob::RenderPageType::Area);
```

Comment:
Ok, let's go for the ```, gpointer user_data)``` that I know well ! Joke aside, I suppose in your response that you mean « pdfjob doesn't need to know meaning if it only provides a method for storing random data », instead of « pdfcanvas…»

I'll switch for an API like suggested before this evening, thank you for your answer.